### DISCUSSION TITLE
loaded addons variable

### DISCUSSION THREAD

**Discussion by AydanL** [Taureon/aps-plus-plus#229] [Suggestions]
Sometimes we want to make an addon for another addon, or just make that an addon is compatible/interactive with another, and then we encounter a difficulty: how to easily tell if an addon is loaded or not? 
The only way currently is to test wether some entity of the addon exists or not, but this can cause problems if the entity can come from another addon that is not the one we want.
So, add a variable that is an array containing the names of all addons already loaded so we can easily do what i described above. As an addon developer it would help alat

**Comment by Frostbyte178** [comment]
global.loadedAddons
array of the loaded addons via their filenames without the .js

**Comment by AydanL** [comment]
oh noice
